omnibus/config/software/chefdk.rb

```diff
@@ -39,16 +39,12 @@
 
 relative_path "chef-dk"
 
-if windows?
-  dependency "ruby-windows"
-  dependency "ruby-windows-devkit"
-else
-  dependency "libffi" if debian?
-  dependency "ruby"
-end
-
-dependency "rubygems"
+dependency "ruby"
+dependency "rubygems-native"
 dependency "bundler"
+# Put this early in the dependency chain to ensure that nokogiri is compiled
+# correctly using the system libraries.
+dependency "nokogiri"
```

Comment:
I could be wrong but I didn't think the order of the deps was a guarantee of the order of execution?